src/System.Net.Ping/tests/FunctionalTests/PingTest.cs

```diff
@@ -122,10 +137,17 @@ public void SendPingWithIPAddress()
                 });
         }
 
-        [Fact]
-        public void SendPingWithIPAddress_AddressAsString()
+        [Theory]
+        [InlineData(AddressFamily.InterNetwork)]
+        [InlineData(AddressFamily.InterNetworkV6)]
+        public void SendPingWithIPAddress_AddressAsString(AddressFamily addressFamily)
         {
             IPAddress localIpAddress = TestSettings.GetLocalIPAddress();
+             if (localIpAddress == null)
```

Comment:
I assume addressFamily should be passed to GetLocalIPAddress here.

Reply:
yes, it is. I don't know why github does not show it.

Reply:
> I don't know why github does not show it

Because it's not there :-)  I just pulled down your change, and it's:
```C#
            IPAddress localIpAddress = TestSettings.GetLocalIPAddress();
             if (localIpAddress == null)
```

Reply:
yes, I found it. I was looking at wrong lines. sorry.

Reply:
:smile:

Reply:
it so happened line 128 has almost exact (and right code). but the broken was on 146. 
Updated and pushed change.